### ISSUE TITLE
Cannot claim route using locomotive cards

### ISSUE THREAD

**Issue by jacobgropp** [byucs340ta/Fall2018#197]
Tried to claim a route using 3 green cards and 2 locomotives and was not given any option to do so.

**Comment by SponJ2000** [comment]
Was it a grey route?

**Comment by jacobgropp** [comment]
Yeah, this works fine. Once again, I wasn't aware of how the claiming of a route worked!
